lq1/accessibility.txt

```diff
@@ -0,0 +1,11 @@
+=========================================
+Useful Console Commands for Accessibility
+=========================================
+
+gl_zfix 1 (to prevent texture flickering aka "z fighting"; only works in Ironwail, but not FTEQCC)
+
+r_dynamic 0 (to disable some flashing lights)
+
+r_waterwarp 0 (to disable the water shader effect)
+
+gl_font calibri (changes the console font to Calibri)
```

Comment:
Which engines support using TrueType fonts like this?

Reply:
I don't have an exhaustive list. I know FTEQW supports it and Ironwail doesn't. Those are the only two engines I've tested so far. Spoike commented that LQ's font is not very readable and suggested this command. I figured it would be a good entry in accessibility.